docs/develop/java/temporal-nexus.mdx

```diff
@@ -0,0 +1,654 @@
+---
+id: nexus
+title: Temporal Nexus - Java SDK Feature Guide
+sidebar_label: Temporal Nexus
+description: Learn how to use Temporal Nexus within the Java SDK to connect Durable Executions within and across Namespaces using a Nexus Endpoint, a Nexus Service contract, and Nexus Operations.
+toc_max_heading_level: 4
+keywords:
+  - temporal nexus java sdk
+
+tags:
+  - temporal-nexus
+  - java-sdk
+  - nexus
+---
+
+Use Temporal Nexus to connect Durable Executions within and across Namespaces using a Nexus Endpoint, a Nexus Service contract, and Nexus Operations.
+
+This page shows how to do the following:
+
+- [Run a development Temporal Service with Nexus enabled](#run-the-temporal-nexus-development-server)
+- [Create caller and handler Namespaces](#create-caller-handler-namespaces)
+- [Create a Nexus Endpoint to route requests from caller to handler](#create-nexus-endpoint)
+- [Define the Nexus Service API contract](#define-nexus-service-api-contract)
+- [Develop a Nexus Service and Operation handlers](#develop-nexus-service-operation-handlers)
+- [Develop a caller Workflow that uses a Nexus Service](#develop-caller-workflow-nexus-service)
+- [Make Nexus calls across Namespaces with a development Server](#nexus-calls-across-namespaces-dev-server)
+- [Make Nexus calls across Namespaces in Temporal Cloud](#nexus-calls-across-namespaces-temporal-cloud)
+
+## Run a development server with Nexus enabled {#run-the-temporal-nexus-development-server}
+
+The first step in working with Temporal Nexus involves starting a Temporal server with Nexus enabled.
+
+Prerequisites:
+
+- [Install the latest Temporal CLI](https://learn.temporal.io/getting_started/java/dev_environment/#set-up-a-local-temporal-service-for-development-with-temporal-cli)
+- [Install the latest Temporal Java SDK](https://learn.temporal.io/getting_started/java/dev_environment/#add-temporal-java-sdk-dependencies) (v1.26.0 or higher)
+
+### Start the Temporal Development Server {#start-the-temporal-development-server}
+
+Start the Temporal Development Server by using the `temporal server start-dev`, with `system.enableNexus=true`.
+The HTTP port is required for Nexus communications.
+
+```
+temporal server start-dev --http-port 7243 --dynamic-config-value system.enableNexus=true
+```
+
+This command automatically starts the Web UI, creates the default Namespace, and uses an in-memory database.
+
+The Temporal Server should be available on `localhost:7233` and the Temporal Web UI should be accessible at [http://localhost:8233](http://localhost:8233).
+
+## Create caller and handler Namespaces {#create-caller-handler-namespaces}
+
+Before setting up Nexus endpoints, create separate Namespaces for the caller and handler.
+
+```
+temporal operator namespace create --namespace my-target-namespace
+temporal operator namespace create --namespace my-caller-namespace
+```
```

Comment:
Would be nice to briefly describe the two roles.

Reply:
Added a brief description below https://github.com/temporalio/documentation/pull/3160/commits/bcb8c78a36e59b076e6248bf44a6c85e8dd65988